src/styles/_shame.scss

```diff
@@ -23,54 +23,60 @@
 
 // Remove boostrap 3 modal scroll bar.
 .modal {
-    overflow-y: auto;
+  overflow-y: auto;
 }
 
 // Toastr fontawesome support.
 #toast-container > .toast {
-    background-image: none !important;
+  background-image: none !important;
 }
+
 #toast-container > .toast:before {
-    position: fixed;
-    font-family: FontAwesome;
-    font-size: 20px;
-    line-height: 20px;
-    float: left;
-    color: #fff;
-    padding-right: 0.5em;
-    margin: auto 0.5em auto -1.5em;
+  color: #fff;
+  float: left;
+  font-family: FontAwesome;
+  font-size: 20px;
+  line-height: 20px;
+  margin: auto .5em auto -1.5em;
```

Comment:
Mhm are you sure, you are running "make styleslint"? It is supposed to use the same configuration as hound.